packages/platform/atoms/event-types/wrappers/EventTypeWebWrapper.tsx

```diff
@@ -89,71 +85,34 @@ const EventAITab = dynamic(() =>
 
 export type EventTypeWebWrapperProps = {
   id: number;
-  isAppDir?: boolean;
 };
 
-// discriminative factor: isAppDir
 type EventTypeAppComponentProp = {
   id: number;
-  isAppDir: true;
   pathname: string;
-  pageRouter: null;
   appRouter: AppRouterInstance;
 };
 
-// discriminative factor: isAppDir
-type EventTypePageComponentProp = {
-  id: number;
-  isAppDir: false;
-  pageRouter: NextPageRouter;
-  pathname: null;
-  appRouter: null;
-};
-
-type EventTypeAppPageComponentProp = EventTypeAppComponentProp | EventTypePageComponentProp;
-
-export const EventTypeWebWrapper = ({ id, isAppDir }: EventTypeWebWrapperProps & { isAppDir?: boolean }) => {
+export const EventTypeWebWrapper = ({ id }: EventTypeWebWrapperProps) => {
   const { data: eventTypeQueryData } = trpc.viewer.eventTypes.get.useQuery({ id });
 
   if (!eventTypeQueryData) return null;
 
-  return isAppDir ? (
-    <EventTypeAppWrapper {...eventTypeQueryData} id={id} />
-  ) : (
-    <EventTypePageWrapper {...eventTypeQueryData} id={id} />
-  );
-};
-
-const EventTypePageWrapper = ({ id, ...rest }: EventTypeSetupProps & { id: number }) => {
-  const router = usePageRouter();
-  return (
-    <EventTypeWeb {...rest} id={id} isAppDir={false} pageRouter={router} pathname={null} appRouter={null} />
-  );
+  return <EventTypeAppWrapper {...eventTypeQueryData} id={id} />;
 };
 
 const EventTypeAppWrapper = ({ id, ...rest }: EventTypeSetupProps & { id: number }) => {
   const pathname = usePathname();
   const router = useAppRouter();
-  return (
-    <EventTypeWeb
-      {...rest}
-      id={id}
-      isAppDir={true}
-      pathname={pathname ?? ""}
-      pageRouter={null}
-      appRouter={router}
-    />
-  );
+  return <EventTypeWeb {...rest} id={id} pathname={pathname ?? ""} appRouter={router} />;
```

Comment:
I think we can shift this logic in EventTypeWebWrapper directly and remove this component completely

Reply:
on it

Reply:
addressed